pkg/apis/eksctl.io/v1alpha5/managed_nodegroup_test.go

```diff
@@ -63,7 +65,7 @@ var _ = Describe("Managed Nodegroup Validation", func() {
 					AMIFamily: DefaultNodeImageFamily,
 				},
 			},
-			errMsg: "overrideBootstrapCommand is required when using a custom AMI",
+			errMsg: fmt.Sprintf("overrideBootstrapCommand is required when using a %s custom AMI", DefaultNodeImageFamily),
```

Comment:
For AL2, this will read:

```
overrideBootstrapCommand is required when using a AmazonLinux2 custom AMI
```

instead of

```
overrideBootstrapCommand is required when using an AmazonLinux2 custom AMI
```

Ubuntu also suffers from the same issue. To avoid this, we can reword it to:

```suggestion
			errMsg: fmt.Sprintf("overrideBootstrapCommand is required when using a custom AMI based on %s", DefaultNodeImageFamily),
```